publisher/publisher.go

```diff
@@ -369,3 +391,37 @@ func CreateTestingSignedSCT(req []string, k *ecdsa.PrivateKey, precert bool, tim
 	jsonSCT, _ := json.Marshal(jsonSCTObj)
 	return jsonSCT
 }
+
+func (pub *Impl) ProbeLogs() {
+	wg := new(sync.WaitGroup)
+	for _, log := range pub.ctLogsCache.LogURIs() {
+		wg.Add(1)
+		go func() {
```

Comment:
CI is failing because this `func()` captures the `log` var. I think it needs to be a parameter passed to this func instead.